composer.json

```diff
@@ -15,9 +15,9 @@
         "source": "https://github.com/VentureCraft/revisionable"
     },
     "require": {
-        "php": ">=5.4.0",
-        "illuminate/support": "~4.0|~5.0|~5.1|^6.0|^7.0|^8.0|^9.0",
-        "laravel/framework": "~5.4|^6.0|^7.0|^8.0|^9.0"
+        "php": ">=8.1",

```

Comment:
Can't merge this, as it would not be backwards compatible.

Laravel 10 support has been merged in though with #420